xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -37,135 +40,129 @@ CEdl::CEdl()
 
 void CEdl::Clear()
 {
-  m_vecEdits.clear();
-  m_vecSceneMarkers.clear();
+  m_edits.clear();
+  m_sceneMarkers.clear();
   m_totalCutTime = 0;
   m_lastEditTime = -1;
 }
 
-bool CEdl::ReadEditDecisionLists(const CFileItem& fileItem, const float fFramesPerSecond)
```

Comment:
any reason to remove the const? I think you do this in a few places. The commit descriptions just says `CodeGuidelines for ReadEditDecisionLists` which doesn't give much context.

Reply:
While the commit message doesn't explain why, the commit itself removes a fixme where the reason is present. Pointed by ksooo sometime ago, the const here for fps is not needed, the callee shouldn't care what this class does with the fps value